### PR TITLE
windows-build-tools

### PR DESCRIPTION
**Pull request by Willyfrog**
added a lighter option to build the libpng dep on windows.

### CODE REVIEW

File: site/content/contribute/desktop/developer-setup.md [hunk 1]
```diff
@@ -51,7 +53,11 @@ If the above command fails, it might be due to not having the right policy to in
 
 ##### pngquant-bin
 
-On Windows there is no default `libpng-dev` available. Currently the best option is to install Visual Studio Community, which provides the dependencies required to build the package.
+On Windows there is no default `libpng-dev` available. There are two options for getting it to build:
+- Install windows build toosl: `npm install windows-build-tools`
```
Comment (justinegeffen) [inline site/content/contribute/desktop/developer-setup.md:57]:
```suggestion
- Install Windows build tools: `npm install windows-build-tools`
```

File: site/content/contribute/desktop/developer-setup.md [hunk 2]
```diff
@@ -51,7 +53,11 @@ If the above command fails, it might be due to not having the right policy to in
...
+- Install Visual Studio Community
+
+Both of which provides the dependencies required to build the package. After one of the two options are installed, install manually: `npm install libpng-dev`
```
Comment (justinegeffen) [inline site/content/contribute/desktop/developer-setup.md:60]:
```suggestion
Either option provides the dependencies required to build the package. After one of the two options are installed, install `npm install libpng-dev` manually. 
```

File: site/content/contribute/desktop/developer-setup.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ subsection: "Desktop App"
 - Node.js 8 or later
 - NPM
 - Git
+- Windows Build Tools or Visual Studio Community edition (see Known Problems section)
```
Comment (justinegeffen) [inline site/content/contribute/desktop/developer-setup.md:14]:
```suggestion
- Windows Build Tools or Visual Studio Community edition (see Known Problems section).
```
Is the "Known Problems" section in this document? And is it a new section?

Comment (Willyfrog) [reply site/content/contribute/desktop/developer-setup.md:14]: yes, it is on line 45

**Comment by Willyfrog** [comment]
@justinegeffen i merged your suggestions (thanks!) and answered your question, unless I missunderstood something. Can you take a second pass?

File: site/content/contribute/desktop/developer-setup.md [hunk 4]
```diff
@@ -11,6 +11,8 @@ subsection: "Desktop App"
 - Node.js 8 or later
 - NPM
 - Git
+- Windows Build Tools or Visual Studio Community edition (see Known Problems section).
+- Recomended: [Chocolatey](https://chocolatey.org/) which we will use to install other deps.
```
Comment (justinegeffen) [inline site/content/contribute/desktop/developer-setup.md:15]:
```suggestion
- Recommended: [Chocolatey](https://chocolatey.org/) which we will use to install other deps.
```

**Pull request merged**